packages/main/src/plugin/index.ts

```
@@ -1942,6 +1942,13 @@ export class PluginSystem {
       },
     );
 
+    this.ipcHandle(
+      'util:createHash',
+      async (_listener, input: string, algorithm: string = 'sha512'): Promise<string> => {
```

Biome (2.1.2)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)